docs/os/tutorials/add_shell.md

```diff
@@ -120,26 +116,46 @@ Once you have a connection set up, you can connect to your device as follows:
 To test and make sure that the Shell is running, first just hit <return>:
     
 ```no-highlight
-3534: >
+004543 compat>
 ```
 
-Remember, we turned the CONSOLE_PROMPT and the CONSOLE_TICKS on earlier. You can try some commands now:
+bletiny app uses legacy shell API to register command callbacks. That's why the prompt says "compat". All the legacy commands are registered to compat module to provide backward compatibility.
```

Comment:
Since we aren't planning to deprecate the shell 1.0 API and we are not recommending one API over another - lets not use "legacy" - I recommend something like this:
1) "bletiny app uses **legacy** shell API to register command callbacks" : Use `the 1.0` instead of `legacy`
2)  "That's why the prompt says "compat". All the legacy commands are registered to compat module to provide backward compatibility" -> Commands registered using the 1.0 shell API are automatically registered to the "compat" module and have "compat" for the prompt.